test/wpt/README.md

```diff
@@ -155,8 +159,16 @@ expected failures.
     // Optional: the test will be skipped with the reason printed
     "skip": "explain why we cannot run a test that's supposed to pass",
 
-    // Optional: the test will be skipped with the reason printed
-    "fail": "explain why we the test is expected to fail"
+    // Optional: failing tests
+    "fail": {
+      "expected": [
```

Comment:
Same as above, and it would be nice to document the `note` field

Reply:
done.